### ISSUE TITLE
When the lod value for some zoom level is 0 tiles is not display properly 

### ISSUE THREAD

**Issue by mvtm-dn** [Esri/esri-leaflet#1380]
### Describe the bug

According to the wrong condition in the case when this._lodMap[zoom]===0 TiledMapLayer uses zoom instead of 0.  And TiledMapLayer tries to load the wrong tiles.

### Reproduction

Alas, but problem ArcGIS tiles with level 0 is private and I can't send link

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 20.04.6 LTS (Focal Fossa)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
    Memory: 9.11 GB / 31.07 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.14.0 - ~/.nvm/versions/node/v16.14.0/bin/node
    npm: 8.3.1 - ~/.nvm/versions/node/v16.14.0/bin/npm
  Browsers:
    Chrome: 117.0.5938.92
  npmPackages:
    leaflet: ^1.6.0 => 1.6.0 
    esri-leaflet: ^3.0.11 => 3.0.11
```

**Comment by gavinr-maps** [comment]
Thank you for logging this issue. I think I was able to find a public tiled layer that has LOD 0 that I can see this issue on. See the steps below - can you please confirm that this is the issue you're reporting?

## Replication case:

1. Open https://jsbin.com/qafojoc/2/edit?html,output which is zoomed to LOD 1
2. Click the zoom out button so you're now zoomed to LOD 0
3. **Expected**: map tiles/repeats across the view area like this:  ![image](https://github.com/Esri/esri-leaflet/assets/137905994/818fb5dc-9340-4549-a5f3-b4cfb7c75ac5)
    - **Actual**: only one tile of the map is seen in the center of the screen, with gray areas to the left/right like this: 
![image](https://github.com/Esri/esri-leaflet/assets/137905994/1573f07c-1047-4e59-81c7-7ed447936e26)

**Comment by mvtm-dn** [comment]
> Thank you for logging this issue. I think I was able to find a public tiled layer that has LOD 0 that I can see this issue on. See the steps below - can you please confirm that this is the issue you're reporting?
> 
> ## Replication case:
> 1. Open https://jsbin.com/qafojoc/2/edit?html,output which is zoomed to LOD 1
> 2. Click the zoom out button so you're now zoomed to LOD 0
> 3. **Expected**: map tiles/repeats across the view area like this:  ![image](https://user-images.githubusercontent.com/137905994/283670529-818fb5dc-9340-4549-a5f3-b4cfb7c75ac5.png)
>    
>    * **Actual**: only one tile of the map is seen in the center of the screen, with gray areas to the left/right like this:
>      ![image](https://user-images.githubusercontent.com/137905994/283670976-1573f07c-1047-4e59-81c7-7ed447936e26.png)

Looks similar

**Comment by gavinr-maps** [comment]
Fixed in [v3.0.12](https://github.com/Esri/esri-leaflet/releases/tag/v3.0.12).
